### PR TITLE
login page done

### PR DESCRIPTION
**Pull request by Adarsh-Patel-Dev**
image

### CODE REVIEW

File: loginPage/login.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    </div>
+
+    <div class="para">
+
+        <div class="signup-page">
+            <h2 class="signup-title">Log In</h2>
+            <input type="text" class="signup-input" placeholder="Email">
+            <input type="text" class="signup-input" placeholder="Password">
+            <div class="checkbox">
+                <input type="checkbox" class="signup" >
```
Comment (pankaj-prog) [inline loginPage/login.html]: why you have given class name sign up here

File: index.css [hunk 2]
```diff
@@ -0,0 +1,250 @@
...
+    border-radius: 2.5rem;
+}
+
+.signup-title {
+    padding: 0rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-style: normal;
+    font-weight: bold;
```
Comment (pankaj-prog) [inline index.css]: header tags are bold by default

**Review by pankaj-prog — CHANGES_REQUESTED** [review]
you should use better class names, which are descriptive. 
current class names are very confusing.

File: loginPage/login.html (hunk 1 above)
Comment (Adarsh-Patel-Dev) [reply loginPage/login.html]: oh Thanks i need to remove it.

File: index.css (hunk 2 above)
Comment (Adarsh-Patel-Dev) [reply index.css]: I didn't know that I'll remove it

File: index.css [hunk 3]
```diff
@@ -0,0 +1,252 @@
...
+.signup-page {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    flex-direction: column;
+    position: absolute;
+    width: 35rem;
+    height: 45rem;
+    left: 50%;
+    top: 60%;
```
Comment (theVivek-kumar) [inline index.css:169]: looks good to me you already resolve that

File: index.css [hunk 4]
```diff
@@ -0,0 +1,252 @@
...
+    border-radius: 2.5rem;
+}
+
+.signup-title {
+    padding: 0rem;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    font-style: normal;
+    font-size: 48px;
```
Comment (theVivek-kumar) [inline index.css]: use rem

File: loginPage/login.html [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+        integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <title>QUANTUM UI | Home</title>
+</head>
+
+<body>
+
+    <!--navigation--container -->
+
+
+    <div class="nav-bar">
```
Comment (theVivek-kumar) [inline loginPage/login.html]: use nav tag

File: loginPage/login.html [hunk 6]
```diff
@@ -0,0 +1,93 @@
...
+        <div class="signup-page">
+            <h2 class="signup-title">Log In</h2>
+            <input type="text" class="signup-input" placeholder="Email">
+            <input type="text" class="signup-input" placeholder="Password">
+            <div class="checkbox">
+                <input type="checkbox" >
+                <label class="signup-label">Remember Me</label>
+            </div>
+
+            <a href="#" class="btn-login">Log In</a>
```
Comment (theVivek-kumar) [inline loginPage/login.html:86]: class name should be login-btn

File: main.css [hunk 7]
```diff
@@ -0,0 +1,54 @@
...
+    --BLUE-50: #5353f1;
+    --BLUE-20: #d9edf7;
+
+    --WHITE: #ffffff;
+    --GRAY-80: #333333;
+    --GREY-60: #929191;
+    --GRAY-40: #8f8e8e;
+    --GRAY-50: #7e7a7a;
+    --GRAY-20: #C4C4C4;
+    --BLACK-100: #000000;
```
Comment (theVivek-kumar) [inline main.css:39]: this looks good to me

**Review by theVivek-kumar — CHANGES_REQUESTED** [review]

**Pull request merged**